### PR TITLE
[CSharp-RichCards] Update to Microsoft.Bot.Builder v3.5.0 and add references to new emulator in README.md

### PR DESCRIPTION
**Pull request by ejadib**
Includes new cards: AnimationCard, VideoCard and AudioCard

### CODE REVIEW

File: CSharp/cards-RichCards/README.md [hunk 1]
```diff
@@ -152,6 +152,7 @@ You will see the following in the Bot Framework Emulator, Facebook Messenger and
 
 To get more information about how to get started in Bot Builder for .NET and Attachments please review the following resources:
 * [Bot Builder for .NET](https://docs.botframework.com/en-us/csharp/builder/sdkreference/index.html)
+* [Adding Attachments to a Message](https://docs.botframework.com/en-us/core-concepts/attachments)
 * [Attachments Property](https://docs.botframework.com/en-us/csharp/builder/sdkreference/activities.html#attachmentsproperty)
 * [Attachments, Cards and Actions](https://docs.botframework.com/en-us/csharp/builder/sdkreference/attachments.html)
```
Comment (pcostantini) [inline CSharp/cards-RichCards/README.md:284]: At L108 it reads:
> For a complete authentication flow sample take a look **to** the [AuthBot](https://github.com/matvelloso/authbot).

Shouldn't it read as:

> For a complete authentication flow sample take a look **at** the [AuthBot](https://github.com/matvelloso/authbot).

**Review by pcostantini — APPROVED** [review]
There may by a grammar error in the Readme file where it links to the AuthBot.
Also, the AuthBot was moved to https://github.com/MicrosoftDX/AuthBot

**Review by pcostantini — CHANGES_REQUESTED** [review]
Please check grammar issue where it links to Authbot,
Also, AuthBot was moved to 
https://github.com/MicrosoftDX/AuthBot

File: CSharp/cards-RichCards/README.md (hunk 1 above)
Comment (ejadib) [reply CSharp/cards-RichCards/README.md:284]: Done

**Review by pcostantini — APPROVED** [review]

File: CSharp/cards-RichCards/CardsDialog.cs [hunk 2]
```diff
@@ -131,5 +139,85 @@ private static Attachment GetSigninCard()
...
+                    }
+                }
+            };
+
+            return videoCard.ToAttachment();
+        }
+
+        private static Attachment GetAudioCard()
+        {
+            var currentUrl = HttpContext.Current.Request.Url;
```
Comment (pcostantini) [inline CSharp/cards-RichCards/CardsDialog.cs]: Unused var

File: CSharp/cards-RichCards/CardsDialog.cs [hunk 3]
```diff
@@ -131,5 +139,85 @@ private static Attachment GetSigninCard()
...
+                    }
+                }
+            };
+
+            return animationCard.ToAttachment();
+        }
+
+        private static Attachment GetVideoCard()
+        {
+            var currentUrl = HttpContext.Current.Request.Url;
```
Comment (pcostantini) [inline CSharp/cards-RichCards/CardsDialog.cs]: Unused var

Comment (ejadib) [reply CSharp/cards-RichCards/CardsDialog.cs]: Removed

File: CSharp/cards-RichCards/CardsDialog.cs (hunk 2 above)
Comment (ejadib) [reply CSharp/cards-RichCards/CardsDialog.cs]: Removed

**Review by pcostantini — APPROVED** [review]

**Pull request merged**